### DISCUSSION TITLE
Maternal Mortality Rate vs Female Life Expectancy for 2020

### DISCUSSION THREAD

**Discussion by matt-humphrey** [has2k1/plotnine#840] [2024 Plotnine Contest]
### Authors

Matt Humphrey

### Links

[linkedin](https://www.linkedin.com/in/matthumphrey97/), [github](https://github.com/matt-humphrey)

### Full description

My first submission to the Plotnine contest!

This is a quick analysis to compare the maternal mortality rate (rate of female deaths due to pregnancy complications) with female life expectancy at birth. The output is an OWID-style scatterplot, categorising and colouring each country by continent. Furthermore, I've chosen to label only a handful of countries on the graph, and adjust the alpha so these particular countries stand out, without hiding the rest.

This example highlights the capabilities of Plotnine, showcasing it's ability to create beautiful graphs that convey a clear message.

![scatterplot of mmr vs fle (2020)](https://github.com/user-attachments/assets/f13644e7-333c-4f52-bbf2-3d2080433ced)

### Code repository

https://github.com/matt-humphrey/mmr-2020/tree/main

**Comment by has2k1** [comment]
![plot (1)](https://github.com/user-attachments/assets/e5b81f7a-e3e7-46d7-95d8-3cf1d82bd280)

**Comment by EconomiCurtis** [comment]
Ahoy @matt-humphrey , 
I am one of the contest organizers. 
If you'd like, we'd be happy to send you a little Plotnine and Posit care package of hex-shaped stickers. Just a small thanks for taking part. https://forms.gle/M8QmM286MSHbkDRd7

**Comment by has2k1** [comment]
@matt-humphrey, The winning submission has been announced [here](https://posit.co/blog/winner-of-the-2024-plotnine-plotting-contest/). Thank you for taking part in the contest.
